src/include/s390/dasd/dialogs.rb

```diff
@@ -368,11 +368,6 @@ def PerformAction(action)
           end
           DASDController.FormatDisks(devices, num_parallel)
```

Comment:
I would write here comment that we intentionally do not active disks with link to that bugzilla, so we know in future why we do not do it and can revisit decision if it is needed.